security/certManager/acme/ocf/certManager.go

```diff
@@ -20,20 +23,32 @@ import (
 
 // Config set configuration.
 type Config struct {
-	acme2.Config
-	DeviceID string `envconfig:"DEVICE_ID" env:"DEVICE_ID" long:"device_id" description:"DeviceID for OCF Identity Certificate"`
+	certManager.Config
+	DeviceID string `envconfig:"ACME_DEVICE_ID" env:"ACME_DEVICE_ID" long:"device_id" description:"DeviceID for OCF Identity Certificate"`
+}
+
+// NewOcfCertManager create new CertManager
+func NewOcfCertManager(config Config) (certManager.CertManager, error) {
+	if config.Type == certManager.FileType {
+		return file.NewCertManagerFromConfiguration(config.File)
+	}
+	if config.Type == certManager.AcmeType {
```

Comment:
I know that there are return statements in both if's, but just for the sake of (possible) future additions, wouldn't it be better to have `else if` instead of just `if` to clearly see that these are two alternatives.

Reply:
Change to else if